### PR TITLE
:sparkles: Dice Presence

### PR DESCRIPTION
**Pull request by TheGamerzs**
image
image
image

Will comment when logo is fixed

### CODE REVIEW

**Comment by TheGamerzs** [comment]
image
image

File: websites/D/Dice Sweden/presence.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  };
+
+  switch (window.location.pathname) {
+    case "/":
+      presenceData.details = "Home";
+      presenceData.startTimestamp = timestamp;
+      presenceData.buttons = [{ label: "Home", url: "https://www.dice.se" }];
```
Comment (SlowLife1661) [inline websites/D/Dice Sweden/presence.ts:25]: Not allowed ( buttons )

File: websites/D/Dice Sweden/presence.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      { label: "View " + gameTitle, url: window.location.href }
+    ];
+  }
+
+  if (presenceData.details == null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else {
+    presence.setActivity(presenceData);
+  }
```
Comment (SlowLife1661) [inline websites/D/Dice Sweden/presence.ts:161]: Never null?

Comment (TheGamerzs) [reply websites/D/Dice Sweden/presence.ts:196]: Removed defualt

File: websites/D/Dice Sweden/presence.ts (hunk 1 above)
Comment (TheGamerzs) [reply websites/D/Dice Sweden/presence.ts]: Removed

File: websites/D/Dice Sweden/presence.ts [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  return window.location.pathname.includes("/news-articles/");
+}
+
+function isGame() {
+  return window.location.pathname.includes("/game/");
+}
+
+function isProfile() {
+  return window.location.pathname.includes("/people/");
+}
```
Comment (Bas950) [inline websites/D/Dice Sweden/presence.ts:16]: you dont need functions for this if its only used once...

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — CHANGES_REQUESTED** [review]
move `startTimestamp` to `presenceData` object.

File: websites/D/Dice Sweden/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  "settings": [
+    {
+      "id": "buttons",
+      "title": "Show Buttons",
+      "icon": "fas fa-compress-arrows-alt",
+      "value": true
+    },
+    {
+      "id": "timestamps",
+      "title": "Show Time Elapsed",
```
Comment (ririxidev) [inline websites/D/Dice Sweden/dist/metadata.json:26]:
```suggestion
      "title": "Show Elapsed Time",
```
😉

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**